src/frontend/js/components/PaymentButton/index.spec.tsx

```diff
@@ -64,13 +67,23 @@ describe.each([
     const Wrapper = ({
       client = createTestQueryClient({ user: true }),
       children,
-    }: PropsWithChildren<{ client?: QueryClient }>) => {
+      product,
+    }: PropsWithChildren<{ client?: QueryClient; product: Product }>) => {
+      const [order, setOrder] = useState<Maybe<Order>>();
+
+      // eslint-disable-next-line react/jsx-no-constructed-context-values
+      const context: SaleTunnelContextType = {
+        product,
+        order,
+        setOrder,
+      };
```

Comment:
Just memoize this value to make eslint happy

Reply:
I agree. Sometimes I think eslint pushes for too much over-optimization leading to verbose code 🙄